src/services/Templates/TemplateApi.ts

```diff
@@ -158,6 +162,19 @@ export const getTemplateSnapshots: (
   return data;
 };
 
+export const getTemplatesForSnapshots: (
+  snapshotUuids: string[],
+) => Promise<TemplateCollectionResponse> = async (snapshotUuids: string[]) => {
+  const { data } = await axios.get<TemplateCollectionResponse>(
+    `/api/content-sources/v1/templates/?${objectToUrlParams({
+      offset: '0',
+      limit: '-1',
```

Comment:
I know this is likely fine for the short-term, but could this cause issues at somepoint?

Reply:
I really don't think so, it would be a large change to how our API works.